pkg/envtest/server.go

```diff
@@ -43,33 +43,56 @@ It's possible to override some defaults, by setting the following environment va
 	KUBEBUILDER_CONTROLPLANE_START_TIMEOUT (string supported by time.ParseDuration): timeout for test control plane to start. Defaults to 20s.
 	KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT (string supported by time.ParseDuration): timeout for test control plane to start. Defaults to 20s.
 	KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT (boolean): if set to true, the control plane's stdout and stderr are attached to os.Stdout and os.Stderr
-
 */
 const (
-	envUseExistingCluster  = "USE_EXISTING_CLUSTER"
-	envKubeAPIServerBin    = "TEST_ASSET_KUBE_APISERVER"
-	envEtcdBin             = "TEST_ASSET_ETCD"
-	envKubectlBin          = "TEST_ASSET_KUBECTL"
-	envKubebuilderPath     = "KUBEBUILDER_ASSETS"
-	envStartTimeout        = "KUBEBUILDER_CONTROLPLANE_START_TIMEOUT"
-	envStopTimeout         = "KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT"
-	envAttachOutput        = "KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT"
-	defaultKubebuilderPath = "/usr/local/kubebuilder/bin"
-	StartTimeout           = 60
-	StopTimeout            = 60
-
+	envUseExistingClusterEnvVar = "USE_EXISTING_CLUSTER"
+	envKubeAPIServerBinEnvVar   = "TEST_ASSET_KUBE_APISERVER"
+	envEtcdBinEnvVar            = "TEST_ASSET_ETCD"
+	envKubectlBinEnvVar         = "TEST_ASSET_KUBECTL"
+	envKubebuilderPathEnvVar    = "KUBEBUILDER_ASSETS"
+	envStartTimeoutEnvVar       = "KUBEBUILDER_CONTROLPLANE_START_TIMEOUT"
+	envStopTimeoutEnvVar        = "KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT"
+	envAttachOutputEnvVar       = "KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT"
```

Comment:
I just renamed the env vars const to make clear what is EnvVar and what is default values